person.rb

```diff
@@ -1,12 +1,17 @@
-class Person
+require './nameable'
+require './capitalize'
+require './trimmer'
+
+class Person < Nameable
```

Comment:
:heavy_check_mark: Good job your application works as expected :clap: :clap: 

![Screenshot from 2023-03-08 02-44-46](https://user-images.githubusercontent.com/16392046/223581203-9c44b35e-867e-4130-9aad-bd5c44bdd61b.png)

Reply:
Hi @Felix45, thanks for the review 🤗